### DISCUSSION TITLE
RPC services not available

### DISCUSSION THREAD

**Discussion by almostheaven1863** [lightningnetwork/lnd#8858] [Troubleshooting]
I just attempted to update my LND to the latest version bc of the bug concern. I have it installed. Now when I attempt to start LND, I get the error RPC services not available. I did not change any settings in my lnd.conf file. so I'm not sure how to fix this error.

**Comment by mohamedawnallah** [comment]
Have you installed the RPCs using `make install tags="signrpc walletrpc routerrpc invoicesrpc ..."`?

As found in the docs here:
https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#installing-lnd-from-source

**Reply by almostheaven1863** [reply]
I figured it out. I had to go back and unlock the wallet. Once I did that, it now is connecting correctly again. Thanks.
